### ISSUE TITLE
Improve contrast for button hover states

### ISSUE THREAD

**Issue by alex-ketch** [stencila/hub#296]
The colour contrast for buttons hover states is not accessible.
They should be updated to be legible.

> ![Kapture 2020-03-10 at 14 15 20](https://user-images.githubusercontent.com/1646307/76345354-970f2980-62d9-11ea-94be-061b4ff8379f.gif)

**Comment by nokome** [comment]
Fixed.
